src/test/java/alinatinca/tests/Tema7/BaseTest.java

```diff
@@ -0,0 +1,29 @@
+package alinatinca.tests.Tema7;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import tincaalina.BrowserManager;
```

Comment:
Please make the import from the right package for this class as you removed tincaalina package. Thanks!

Reply:
done